notebooks/membership_inference_attack.ipynb

```diff
@@ -76,16 +184,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "attack.attack_output(x_train[:100], y_train[:100], np.ones((100,)))"
+    "output = attack.attack_output(\n",
+    "    x_train[:100], \n",
+    "    y_train[:100], \n",
+    "    np.ones((100,))\n",
+    ")\n",
+    "\n",
+    "output.to_json()"
```

Comment:
Actualy when i run the notebook locally I get the error `AttributeError: 'UserOutputInferenceAttack' object has no attribute 'to_json'`. strangely it doses work on google colab and also if I just change it to `print(output)`.
Does someone has a explanation for that?
also throws that error, when trying to call `.to_dict()`

Reply:
Maybe you have an outdated version of our package locally which the notebook uses somehow? Try uninstalling it and then run the notebook.

Reply:
ah that actualy resolved it thanks. i just installed the new package over the old one, but uninstalling and reinstaling works